frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -91,13 +107,45 @@ function convertInput(paramStr: string, paramType: ParameterType_ParameterTypeEn
   }
 }
 
+function checkInput(
```

Comment:
The function name `checkInput` is ambiguous, how do you think about a better name so people can understand what this function does by reading the function name?

Reply:
How about `isValidInput` or `checkInputIsValid`?

Reply:
Both names sound good! But with the conversation in https://github.com/kubeflow/pipelines/pull/8003#discussion_r918495019, we are going to return errorMessage only in this function. So to take that factor into consider, how about function name `generateInputValidationErrorMessage()`?